Description: Newer ffmpeg versions require the input of -b and -ab to be
  in k iso kb. The winff code is already adjusted since revision 480, but
  the preset files were never updated.
Author: Paul Gevers <paul@climbing.nl>
Bug: http://code.google.com/p/winff/issues/detail?id=109

Index: winff-git/presets.xml
===================================================================
--- winff-git.orig/presets.xml	2011-02-18 19:57:11.009589002 +0100
+++ winff-git/presets.xml	2011-02-18 20:10:00.165588998 +0100
@@ -2,13 +2,13 @@
 <presets>
   <BlackberryCurvefs>
     <label>Blackberry Curve Fullscreen</label>
-    <params>-f mp4 -vcodec mpeg4 -b 400k -r 24 -s 320x240 -aspect 4:3 -acodec libfaac -ar 22050 -ac 2 -ab 48kb</params>
+    <params>-f mp4 -vcodec mpeg4 -b 400k -r 24 -s 320x240 -aspect 4:3 -acodec libfaac -ar 22050 -ac 2 -ab 48k</params>
     <extension>mp4</extension>
     <category>Blackberry</category>
   </BlackberryCurvefs>
   <BlackberryCurvews>
     <label>Blackberry Curve Widescreen</label>
-    <params>-f mp4 -vcodec mpeg4 -b 400kb -r 24 -s 320x180 -aspect 16:9 -acodec libfaac -ar 22050 -ac 2 -ab 48kb</params>
+    <params>-f mp4 -vcodec mpeg4 -b 400k -r 24 -s 320x180 -aspect 16:9 -acodec libfaac -ar 22050 -ac 2 -ab 48k</params>
     <extension>mp4</extension>
     <category>Blackberry</category>
   </BlackberryCurvews>
@@ -20,157 +20,157 @@
   </CDWavStereo>
   <GigabeatFS>
     <label>RB Toshiba Gigabeat F/X Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 320x240 -b 600kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 320x240 -b 600k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </GigabeatFS>
   <GigabeatWS>
     <label>RB Toshiba Gigabeat F/X Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 320x176 -b 600kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 320x176 -b 600k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </GigabeatWS>
   <H1020FS>
     <label>RB iRiver H10 20GB Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 160x128 -b 224kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 160x128 -b 224k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H1020FS>
   <H1020WS>
     <label>RB iRiver H10 20GB Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 160x96 -b 224kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 160x96 -b 224k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H1020WS>
   <H105FS>
     <label>RB iRiver H10 5/6GB Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 128x96 -b 224kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 128x96 -b 224k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H105FS>
   <H105WS>
     <label>RB iRiver H10 5/6GB Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 128x80 -b 224kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 128x80 -b 224k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H105WS>
   <H300FS>
     <label>RB iRiver H300 Fullscreen</label>
-    <params>-acodec libmp3lame -ab 96kb -ar 44100 -vcodec mpeg2video -s 224x176 -b 224kb -r 10 -strict -1</params>
+    <params>-acodec libmp3lame -ab 96k -ar 44100 -vcodec mpeg2video -s 224x176 -b 224k -r 10 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H300FS>
   <H300WS>
     <label>RB iRiver H300 Widescreen</label>
-    <params>-acodec libmp3lame -ab 96kb -ar 44100 -vcodec mpeg2video -s 224x128 -b 256kb -r 15 -strict -1</params>
+    <params>-acodec libmp3lame -ab 96k -ar 44100 -vcodec mpeg2video -s 224x128 -b 256k -r 15 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </H300WS>
   <I5GFS>
     <label>RB Apple iPod Video Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 320x240 -b 400kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 320x240 -b 400k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </I5GFS>
   <I5GWS>
     <label>RB Apple iPod Video Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 320x176 -b 400kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 320x176 -b 400k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </I5GWS>
   <M4a>
     <label>MPEG4 Audio</label>
-    <params>-vn -acodec libfaac -ab 112kb -ac 2 -ar 48000</params>
+    <params>-vn -acodec libfaac -ab 112k -ac 2 -ar 48000</params>
     <extension>m4a</extension>
     <category>Audio</category>
   </M4a>
   <NeurosOSDTVHQNTSC>
     <label>Neuros HQ NTSC Fullscreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libxvid -s 640x480 -aspect 4:3 -maxrate 2500kb -b 2000kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libxvid -s 640x480 -aspect 4:3 -maxrate 2500kb -b 2000k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128k -ac 2</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVHQNTSC>
   <NeurosOSDTVHQPAL>
     <label>Neuros HQ PAL Fullscreen</label>
-    <params>-f mp4 -r 25 -vcodec libxvid -s 640x480 -aspect 4:3 -maxrate 2500kb -b 2000kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f mp4 -r 25 -vcodec libxvid -s 640x480 -aspect 4:3 -maxrate 2500kb -b 2000k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128k -ac 2</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVHQPAL>
   <NeurosOSDTVSFNTSC>
     <label>Neuros Small File NTSC Fullscreen</label>
-    <params>-f mp4 -b 800kb -r 29.97 -s 320x240 -aspect 4:3 -vcodec libxvid -ar 48000 -ab 80kb -ac 2 -acodec libfaac</params>
+    <params>-f mp4 -b 800k -r 29.97 -s 320x240 -aspect 4:3 -vcodec libxvid -ar 48000 -ab 80k -ac 2 -acodec libfaac</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVSFNTSC>
   <NeurosOSDTVSFPAL>
     <label>Neuros Small File PAL Fullscreen</label>
-    <params>-f mp4 -b 800kb -r 25 -s 320x240 -aspect 4:3 -vcodec libxvid -ar 48000 -ab 80kb -ac 2 -acodec libfaac</params>
+    <params>-f mp4 -b 800k -r 25 -s 320x240 -aspect 4:3 -vcodec libxvid -ar 48000 -ab 80k -ac 2 -acodec libfaac</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVSFPAL>
   <NeurosOSDTVWSHQNTSC>
     <label>Neuros HQ NTSC Widescreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libxvid -s 704x384 -aspect 16:9 -maxrate 3000kb -b 2500kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libxvid -s 704x384 -aspect 16:9 -maxrate 3000kb -b 2500k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128k -ac 2</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVWSHQNTSC>
   <NeurosOSDTVWSHQPAL>
     <label>Neuros HQ PAL Widescreen</label>
-    <params>-f mp4 -r 25 -vcodec libxvid -s 704x384 -aspect 16:9 -maxrate 3000kb -b 2500kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f mp4 -r 25 -vcodec libxvid -s 704x384 -aspect 16:9 -maxrate 3000kb -b 2500k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 48000 -ab 128k -ac 2</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVWSHQPAL>
   <NeurosOSDTVWSSFNTSC>
     <label>Neuros Small File NTSC Widescreen</label>
-    <params>-f mp4 -b 800kb -r 29.97 -s 352x240 -aspect 16:9 -vcodec libxvid -ar 48000 -ab 80kb -ac 2 -acodec libfaac</params>
+    <params>-f mp4 -b 800k -r 29.97 -s 352x240 -aspect 16:9 -vcodec libxvid -ar 48000 -ab 80k -ac 2 -acodec libfaac</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVWSSFNTSC>
   <NeurosOSDTVWSSFPAL>
     <label>Neuros Small File PAL Widescreen</label>
-    <params>-f mp4 -b 800kb -r 25 -s 352x240 -aspect 16:9 -vcodec libxvid -ar 48000 -ab 80kb -ac 2 -acodec libfaac</params>
+    <params>-f mp4 -b 800k -r 25 -s 352x240 -aspect 16:9 -vcodec libxvid -ar 48000 -ab 80k -ac 2 -acodec libfaac</params>
     <extension>mp4</extension>
     <category>Neuros OSD</category>
   </NeurosOSDTVWSSFPAL>
   <PalmXviDFS>
     <label>Palm Fullscreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libxvid -s 428x320 -aspect 4:3 -maxrate 550kb -b 500kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 44100 -ab 80kb -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libxvid -s 428x320 -aspect 4:3 -maxrate 550kb -b 500k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 44100 -ab 80k -ac 2</params>
     <extension>mp4</extension>
     <category>Palm</category>
   </PalmXviDFS>
   <PalmXviDWS>
     <label>Palm Widescreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libxvid -s 480x320 -aspect 3:2 -maxrate 450kb -b 430kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 44100 -ab 80kb -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libxvid -s 480x320 -aspect 3:2 -maxrate 450kb -b 430k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libfaac -ar 44100 -ab 80k -ac 2</params>
     <extension>mp4</extension>
     <category>Palm</category>
   </PalmXviDWS>
   <PhotoFS>
     <label>RB Apple iPod Photo/Color Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 224x176 -b 320kb -r 15 -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 224x176 -b 320k -r 15 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </PhotoFS>
   <PhotoWS>
     <label>RB Apple iPod Photo/Color Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 224x128 -b 320kb -r 24 -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 224x128 -b 320k -r 24 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </PhotoWS>
   <X5FS>
     <label>RB Cowon iAudio X5 Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 160x128 -b 176kb -r 15 -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 160x128 -b 176k -r 15 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </X5FS>
   <X5WS>
     <label>RB Cowon iAudio X5 Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 160x96 -b 224kb -r 24 -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 160x96 -b 224k -r 24 -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </X5WS>
   <XviDAVIFS>
     <label>XviD FullScreen</label>
-    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 640x480 -aspect 4:3 -maxrate 1800kb -b 1500kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libmp3lame -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 640x480 -aspect 4:3 -maxrate 1800kb -b 1500k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libmp3lame -ar 48000 -ab 128k -ac 2</params>
     <extension>avi</extension>
     <category>AVI</category>
   </XviDAVIFS>
@@ -182,61 +182,61 @@
   </XviDAVIWS>
   <XviDAVIWSAna>
     <label>XviD Widescreen Anamorphic</label>
-    <params>-f avi -r 29.97 -croptop 58 -cropbottom 62 -vcodec libxvid -vtag XVID -s 640x272 -aspect 2.35 -maxrate 1800kb -b 1500kb -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libmp3lame -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f avi -r 29.97 -croptop 58 -cropbottom 62 -vcodec libxvid -vtag XVID -s 640x272 -aspect 2.35 -maxrate 1800kb -b 1500k -qmin 3 -qmax 5 -bufsize 4096 -mbd 2 -bf 2 -flags +4mv -trellis -aic -cmp 2 -subcmp 2 -g 300 -acodec libmp3lame -ar 48000 -ab 128k -ac 2</params>
     <extension>avi</extension>
     <category>AVI</category>
   </XviDAVIWSAna>
   <XviDAVIZENFS>
     <label>Zen Fullscreen</label>
-    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 320x240 -aspect 4:3 -maxrate 1800kb -b 1500k  -acodec libmp3lame -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 320x240 -aspect 4:3 -maxrate 1800kb -b 1500k  -acodec libmp3lame -ar 48000 -ab 128k -ac 2</params>
     <extension>avi</extension>
     <category>Creative Zen</category>
   </XviDAVIZENFS>
   <XviDAVIZENWS>
     <label>Zen Widescreen</label>
-    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 320x240 -aspect 16:9 -maxrate 1800kb -b 1500kb  -acodec libmp3lame -ar 48000 -ab 128kb -ac 2</params>
+    <params>-f avi -r 29.97 -vcodec libxvid -vtag XVID -s 320x240 -aspect 16:9 -maxrate 1800kb -b 1500k  -acodec libmp3lame -ar 48000 -ab 128k -ac 2</params>
     <extension>avi</extension>
     <category>Creative Zen</category>
   </XviDAVIZENWS>
   <ac3dvd192>
     <label>Ac3 DVD - 192kbps Stereo</label>
-    <params>-f ac3 -acodec ac3 -ab 192kb -ar 48000 -ac 2</params>
+    <params>-f ac3 -acodec ac3 -ab 192k -ar 48000 -ac 2</params>
     <extension>ac3</extension>
     <category>Audio</category>
   </ac3dvd192>
   <ac3dvd384>
     <label>Ac3 DVD - 384kbps Stereo</label>
-    <params>-f ac3 -acodec ac3 -ab 384kb -ar 48000 -ac 2</params>
+    <params>-f ac3 -acodec ac3 -ab 384k -ar 48000 -ac 2</params>
     <extension>ac3</extension>
     <category>Audio</category>
   </ac3dvd384>
   <blackberryfs>
     <label>Blackberry Video Fullscreen</label>
-    <params>-f mp4 -r 24 -vcodec libx264 -s 240x180 -aspect 4:3 -maxrate 800kb -bufsize 80k -b 400kb -acodec libfaac -ar 44100 -ab 80kb -ac 2</params>
+    <params>-f mp4 -r 24 -vcodec libx264 -s 240x180 -aspect 4:3 -maxrate 800kb -bufsize 80k -b 400k -acodec libfaac -ar 44100 -ab 80k -ac 2</params>
     <extension>mp4</extension>
     <category>Blackberry</category>
   </blackberryfs>
   <blackberrymp3>
     <label>Blackberry Music</label>
-    <params>-acodec libmp3lame -ab 160kb -ac 2 -ar 44100</params>
+    <params>-acodec libmp3lame -ab 160k -ac 2 -ar 44100</params>
     <extension>mp3</extension>
     <category>Blackberry</category>
   </blackberrymp3>
   <blackberryws>
     <label>Blackberry Video Widescreen</label>
-    <params>-f mp4 -r 24 -vcodec libx264 -s 320x180 -aspect 16:9 -maxrate 800kb -bufsize 80k -b 400kb -acodec libfaac -ar 44100 -ab 80kb -ac 2</params>
+    <params>-f mp4 -r 24 -vcodec libx264 -s 320x180 -aspect 16:9 -maxrate 800kb -bufsize 80k -b 400k -acodec libfaac -ar 44100 -ab 80k -ac 2</params>
     <extension>mp4</extension>
     <category>Blackberry</category>
   </blackberryws>
   <cdma3g>
     <label>CDMA Phone Audio (3g2)</label>
-    <params>-f 3g2 -ar 22050 -ab 128kb -acodec libfaac -s qcif -r 14.985 -vn</params>
+    <params>-f 3g2 -ar 22050 -ab 128k -acodec libfaac -s qcif -r 14.985 -vn</params>
     <extension>3g2</extension>
     <category>Mobile Phones</category>
   </cdma3g>
   <divx>
     <label>MS Compatible AVI</label>
-    <params>-acodec libmp3lame -vcodec msmpeg4 -ab 192kb -b 1000kb -s 640x480 -ar 44100</params>
+    <params>-acodec libmp3lame -vcodec msmpeg4 -ab 192k -b 1000k -s 640x480 -ar 44100</params>
     <extension>avi</extension>
     <category>AVI</category>
   </divx>
@@ -254,37 +254,37 @@
   </dvpal>
   <e200FS>
     <label>RB Sandisk Sansa e200 Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 224x176 -b 320kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 224x176 -b 320k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </e200FS>
   <e200WS>
     <label>RB Sandisk Sansa e200 Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 224x128 -b 320kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 224x128 -b 320k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </e200WS>
   <flvwebFS>
     <label>Flash Video (flv) for Web use Fullscreen</label>
-    <params>-vcodec flv -f flv -r 29.97 -s 320x240 -aspect 4:3 -b 300kb -g 160 -cmp dct  -subcmp dct  -mbd 2 -flags +aic+cbp+mv0+mv4 -trellis 1 -ac 1 -ar 22050 -ab 56kb</params>
+    <params>-vcodec flv -f flv -r 29.97 -s 320x240 -aspect 4:3 -b 300k -g 160 -cmp dct  -subcmp dct  -mbd 2 -flags +aic+cbp+mv0+mv4 -trellis 1 -ac 1 -ar 22050 -ab 56k</params>
     <extension>flv</extension>
     <category>Websites</category>
   </flvwebFS>
   <flvwebWS>
     <label>Flash: Video (flv) for Web use Widescreen</label>
-    <params>-vcodec flv -f flv -r 29.97 -s 320x180 -aspect 16:9 -b 300kb -g 160 -cmp dct -subcmp dct -mbd 2 -flags +aic+cbp+mv0+mv4 -trellis 1 -ac 1 -ar 22050 -ab 56kb</params>
+    <params>-vcodec flv -f flv -r 29.97 -s 320x180 -aspect 16:9 -b 300k -g 160 -cmp dct -subcmp dct -mbd 2 -flags +aic+cbp+mv0+mv4 -trellis 1 -ac 1 -ar 22050 -ab 56k</params>
     <extension>flv</extension>
     <category>Websites</category>
   </flvwebWS>
   <iPodiTunesSmallCRF21FS>
     <label>iPod Small Fullscreen</label>
-    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x240 -aspect 4:3 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x240 -aspect 4:3 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesSmallCRF21FS>
   <iPodiTunesSmallCRF21WS>
     <label>iPod Small WideScreen</label>
-    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x176 -aspect 16:9 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x176 -aspect 16:9 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesSmallCRF21WS>
@@ -296,43 +296,43 @@
   </iPodiTunesSmallCRF21WSAna>
   <iPodiTunesSmallCRF21WSLBFix>
     <label>iPod Small Converted to WideScreen</label>
-    <params>-r 29.97 -croptop 64 -cropbottom 64 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x176 -aspect 16:9 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -croptop 64 -cropbottom 64 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 320x176 -aspect 16:9 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesSmallCRF21WSLBFix>
   <iPodiTunesTVOutCRF21FS>
     <label>iPod TV Out Fullscreen</label>
-    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 512x384 -aspect 4:3 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 512x384 -aspect 4:3 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesTVOutCRF21FS>
   <iPodiTunesTVOutCRF21WS>
     <label>iPod TV Out Widescreen</label>
-    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 624x352 -aspect 16:9 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 624x352 -aspect 16:9 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesTVOutCRF21WS>
   <iPodiTunesTVOutCRF21WSAna>
     <label>iPod TV Out Widescreen Anamorphic</label>
-    <params>-r 29.97 -croptop 60 -cropbottom 60 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 640x272 -aspect 2.35 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -croptop 60 -cropbottom 60 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 640x272 -aspect 2.35 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesTVOutCRF21WSAna>
   <iPodiTunesTVOutCRF21WSLBFix>
     <label>iPod TV Out Converted to Widescreen</label>
-    <params>-r 29.97 -croptop 64 -cropbottom 64 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 640x352 -aspect 16:9 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -croptop 64 -cropbottom 64 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 640x352 -aspect 16:9 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesTVOutCRF21WSLBFix>
   <iPodiTunesIphoneWS>
     <label>iPhone WideScreen</label>
-    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 480x320 -aspect 16:9 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-r 29.97 -vcodec libx264 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -crf 21 -bt 256k -refs 1 -coder 0 -me_method full -me_range 16 -subq 5 -partitions +parti4x4+parti8x8+partp8x8 -g 250 -keyint_min 25 -level 30 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -s 480x320 -aspect 16:9 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>m4v</extension>
     <category>iPod-iTunes</category>
   </iPodiTunesIphoneWS>
   <lgchocolate>
     <label>LG Chocolate</label>
-    <params>-f mp4 -s 240x192 -r 11.988 -b 192kb -ab 56kb -vcodec libxvid -acodec libfaac -ar 22050</params>
+    <params>-f mp4 -s 240x192 -r 11.988 -b 192k -ab 56k -vcodec libxvid -acodec libfaac -ar 22050</params>
     <extension>mp4</extension>
     <category>LG</category>
   </lgchocolate>
@@ -344,55 +344,55 @@
   </lgdare>
   <mp3>
     <label>MP3</label>
-    <params>-acodec libmp3lame -ab 160kb -ac 2 -ar 44100</params>
+    <params>-acodec libmp3lame -ab 160k -ac 2 -ar 44100</params>
     <extension>mp3</extension>
     <category>Audio</category>
   </mp3>
   <mp3m>
     <label>MP3 (Mono)</label>
-    <params>-acodec libmp3lame -ab 32kb -ac 1 -ar 22050</params>
+    <params>-acodec libmp3lame -ab 32k -ac 1 -ar 22050</params>
     <extension>mp3</extension>
     <category>Mobile Phones</category>
   </mp3m>
   <nanoFS>
     <label>RB Apple iPod Nano Fullscreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 176x128 -b 256kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 176x128 -b 256k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </nanoFS>
   <nanoWS>
     <label>RB Apple iPod Nano Widescreen</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg2video -s 176x128 -b 256kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg2video -s 176x128 -b 256k -strict -1</params>
     <extension>mpg</extension>
     <category>Rockbox</category>
   </nanoWS>
   <ntscdvdgoodqFS>
     <label>NTSC DVD Fullscreen</label>
-    <params>-f dvd -vcodec mpeg2video -r 29.97 -s 352x480 -aspect 4:3 -b 4000kb -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192kb -ar 48000 -ac 2</params>
+    <params>-f dvd -vcodec mpeg2video -r 29.97 -s 352x480 -aspect 4:3 -b 4000k -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192k -ar 48000 -ac 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </ntscdvdgoodqFS>
   <ntscdvdgoodqWS>
     <label>NTSC DVD Widescreen</label>
-    <params>-f dvd -vcodec mpeg2video -r 29.97 -s 352x480 -aspect 16:9 -b 4000kb -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192kb -ar 48000 -ac 2</params>
+    <params>-f dvd -vcodec mpeg2video -r 29.97 -s 352x480 -aspect 16:9 -b 4000k -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192k -ar 48000 -ac 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </ntscdvdgoodqWS>
   <ntscdvdhqFS>
     <label>NTSC DVD HQ Fullscreen</label>
-    <params>-f dvd -target ntsc-dvd -r 29.97 -s 720x480 -aspect 4:3 -b 8000kb -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
+    <params>-f dvd -target ntsc-dvd -r 29.97 -s 720x480 -aspect 4:3 -b 8000k -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </ntscdvdhqFS>
   <ntscdvdhqWS>
     <label>NTSC DVD HQ Widescreen</label>
-    <params>-f dvd -target ntsc-dvd -r 29.97 -s 720x480 -aspect 16:9 -b 8000kb -g 12 -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
+    <params>-f dvd -target ntsc-dvd -r 29.97 -s 720x480 -aspect 16:9 -b 8000k -g 12 -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </ntscdvdhqWS>
   <ntscdvdlq>
     <label>NTSC DVD Fast (LQ)</label>
-    <params>-f dvd -target ntsc-dvd -b 5000kb -r 29.97 -s 720x480 -ar 48000 -ab 384kb</params>
+    <params>-f dvd -target ntsc-dvd -b 5000k -r 29.97 -s 720x480 -ar 48000 -ab 384k</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </ntscdvdlq>
@@ -410,31 +410,31 @@
   </ogg1>
   <paldvdgoodqFS>
     <label>PAL DVD Fullscreen</label>
-    <params>-f dvd -vcodec mpeg2video -r 25.00 -s 352x576  -aspect 4:3 -b 4000kb -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192kb -ar 48000 -ac 2</params>
+    <params>-f dvd -vcodec mpeg2video -r 25.00 -s 352x576  -aspect 4:3 -b 4000k -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192k -ar 48000 -ac 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </paldvdgoodqFS>
   <paldvdgoodqWS>
     <label>PAL DVD Widescreen</label>
-    <params>-f dvd -vcodec mpeg2video -r 25.00 -s 352x576 -aspect 16:9 -b 4000kb -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192kb -ar 48000 -ac 2</params>
+    <params>-f dvd -vcodec mpeg2video -r 25.00 -s 352x576 -aspect 16:9 -b 4000k -mbd rd -trellis -mv0 -cmp 2 -subcmp 2 -acodec mp2 -ab 192k -ar 48000 -ac 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </paldvdgoodqWS>
   <paldvdhqFS>
     <label>PAL DVD HQ Fullscreen</label>
-    <params>-f dvd -target pal-dvd -aspect 4:3 -b 8000kb -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
+    <params>-f dvd -target pal-dvd -aspect 4:3 -b 8000k -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </paldvdhqFS>
   <paldvdhqWS>
     <label>PAL DVD HQ Widescreen</label>
-    <params>-f dvd -target pal-dvd -aspect 16:9 -b 8000kb -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
+    <params>-f dvd -target pal-dvd -aspect 16:9 -b 8000k -mbd rd -trellis -mv0 -cmp 0 -subcmp 2</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </paldvdhqWS>
   <paldvdlq>
     <label>PAL DVD Fast (LQ)</label>
-    <params>-f dvd -target pal-dvd -b 5000kb -r 25 -s 720x576 -ar 48000 -ab 384kb</params>
+    <params>-f dvd -target pal-dvd -b 5000k -r 25 -s 720x576 -ar 48000 -ab 384k</params>
     <extension>mpg</extension>
     <category>DVD</category>
   </paldvdlq>
@@ -446,49 +446,49 @@
   </palvcdhq>
   <ps3720p>
     <label>PS3 720p Widescreen</label>
-    <params>-f mp4 -r 23.976 -vcodec libx264 -s 1280x720 -b 3000kb -aspect 16:9 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0  -maxrate 3500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 384kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 23.976 -vcodec libx264 -s 1280x720 -b 3000k -aspect 16:9 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0  -maxrate 3500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 384k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>PS3</category>
   </ps3720p>
   <ps3720pfs>
     <label>PS3 720p Fullscreen</label>
-    <params>-f mp4 -r 23.976 -vcodec libx264 -s 1280x720 -b 3000kb -aspect 4:3 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0  -maxrate 3500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 384kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 23.976 -vcodec libx264 -s 1280x720 -b 3000k -aspect 4:3 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0  -maxrate 3500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 384k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>PS3</category>
   </ps3720pfs>
   <qmov>
     <label>Quicktime MOV</label>
-    <params>-f mov -acodec libfaac -b 1250kb -r 25 -ab 128kb -s 640x480 -ac 2 -ar 48000</params>
+    <params>-f mov -acodec libfaac -b 1250k -r 25 -ab 128k -s 640x480 -ac 2 -ar 48000</params>
     <extension>mov</extension>
     <category>QuickTime</category>
   </qmov>
   <wma>
     <label>WMA</label>
-    <params>-vn  -acodec wmav2 -ab 160kb</params>
+    <params>-vn  -acodec wmav2 -ab 160k</params>
     <extension>wma</extension>
     <category>Audio</category>
   </wma>
   <wmv>
     <label>WMV2 Generic</label>
-    <params>-vcodec wmv2  -acodec wmav2 -b 1000kb -ab 160kb -r 25</params>
+    <params>-vcodec wmv2  -acodec wmav2 -b 1000k -ab 160k -r 25</params>
     <extension>wmv</extension>
     <category>WMV</category>
   </wmv>
   <wmv2broadband>
     <label>WMV for Web Use</label>
-    <params>-vcodec wmv2  -acodec wmav2 -b 640kb -ab 128kb -r 29.97 -s 320x240</params>
+    <params>-vcodec wmv2  -acodec wmav2 -b 640k -ab 128k -r 29.97 -s 320x240</params>
     <extension>wmv</extension>
     <category>WMV</category>
   </wmv2broadband>
   <wmv2xbox360>
     <label>Xbox 360</label>
-    <params>-vcodec wmv2  -acodec wmav2 -b 1200kb -ab 160kb -r 25 -s 720x480</params>
+    <params>-vcodec wmv2  -acodec wmav2 -b 1200k -ab 160k -r 25 -s 720x480</params>
     <extension>wmv</extension>
     <category>WMV</category>
   </wmv2xbox360>
   <wmv2zune>
     <label>Zune</label>
-    <params>-vcodec wmv2  -acodec wmav2 -b 640kb -ab 128kb -r 23.97 -s 320x240</params>
+    <params>-vcodec wmv2  -acodec wmav2 -b 640k -ab 128k -r 23.97 -s 320x240</params>
     <extension>wmv</extension>
     <category>WMV</category>
   </wmv2zune>
@@ -500,25 +500,25 @@
   </wmv4powerpoint>
   <x264HQFS>
     <label>MP4 Fullscreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libx264 -s 640x480 -b 1000kb -aspect 4:3 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -b 1250k -maxrate 1500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libx264 -s 640x480 -b 1000k -aspect 4:3 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -b 1250k -maxrate 1500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>MPEG-4</category>
   </x264HQFS>
   <x264HQPSP169robertswain>
     <label>PSP Widescreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libx264 -s 480x272 -aspect 16:9 -b 1250kb -maxrate 4M -bufsize 4M -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method umh -subq 6 -trellis 1 -refs 2 -bf 1 -coder 1 -me_range 16 -g 300 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -bt 1250k -qcomp 0.6 -qmin 10 -qmax 51 -qdiff 4 -level 21 -acodec libfaac -ab 128kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libx264 -s 480x272 -aspect 16:9 -b 1250k -maxrate 4M -bufsize 4M -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method umh -subq 6 -trellis 1 -refs 2 -bf 1 -coder 1 -me_range 16 -g 300 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -bt 1250k -qcomp 0.6 -qmin 10 -qmax 51 -qdiff 4 -level 21 -acodec libfaac -ab 128k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>PSP</category>
   </x264HQPSP169robertswain>
   <x264HQPSP43>
     <label>PSP Fullscreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libx264 -s 640x480 -aspect 4:3 -b 1250kb -maxrate 4M -bufsize 4M -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method umh -subq 6 -trellis 1 -refs 2 -bf 1 -coder 1 -me_range 16 -g 300 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -bt 1250k -qcomp 0.6 -qmin 10 -qmax 51 -qdiff 4 -level 21 -acodec libfaac -ab 128kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libx264 -s 640x480 -aspect 4:3 -b 1250k -maxrate 4M -bufsize 4M -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method umh -subq 6 -trellis 1 -refs 2 -bf 1 -coder 1 -me_range 16 -g 300 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -bt 1250k -qcomp 0.6 -qmin 10 -qmax 51 -qdiff 4 -level 21 -acodec libfaac -ab 128k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>PSP</category>
   </x264HQPSP43>
   <x264HQWS>
     <label>MP4 Widescreen</label>
-    <params>-f mp4 -r 29.97 -vcodec libx264 -s 704x384 -b 1000kb -aspect 16:9 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -b 1250k -maxrate 1500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 112kb -ar 48000 -ac 2</params>
+    <params>-f mp4 -r 29.97 -vcodec libx264 -s 704x384 -b 1000k -aspect 16:9 -flags +loop -cmp +chroma -deblockalpha 0 -deblockbeta 0 -b 1250k -maxrate 1500k -bufsize 4M -bt 256k -refs 1 -bf 3 -coder 1 -me_method umh -me_range 16 -subq 7 -partitions +parti4x4+parti8x8+partp8x8+partb8x8 -g 250 -keyint_min 25 -level 30 -qmin 10 -qmax 51 -qcomp 0.6 -trellis 2 -sc_threshold 40 -i_qfactor 0.71 -acodec libfaac -ab 112k -ar 48000 -ac 2</params>
     <extension>mp4</extension>
     <category>MPEG-4</category>
   </x264HQWS>
@@ -536,13 +536,13 @@
   </android2>
   <android3>
     <label>Google Android video. VHQ 16:9 widescreen</label>
-    <params>-s 480x272 -vcodec libx264 -b 500k -threads 0 -acodec libfaac -ab 96kb</params>
+    <params>-s 480x272 -vcodec libx264 -b 500k -threads 0 -acodec libfaac -ab 96k</params>
     <extension>mp4</extension>
     <category>Google Android</category>
   </android3>
   <android4>
     <label>Google Android video. VHQ 4:3 fullscreen</label>
-    <params>-s 432x320 -vcodec libx264 -b 500k -threads 0 -acodec libfaac -ab 96kb</params>
+    <params>-s 432x320 -vcodec libx264 -b 500k -threads 0 -acodec libfaac -ab 96k</params>
     <extension>mp4</extension>
     <category>Google Android</category>
   </android4>
@@ -572,91 +572,91 @@
   </N810WS2>
   <NokiaN73N81N95WS>
     <label>Nokia Nxx Video WideScreen</label>
-    <params>-f mp4 -r 15 -vcodec mpeg4 -s 320x240 -b 320kb -aspect 16:9 -acodec libfaac -ab 96kb -ar 44100 -ac 2</params>
+    <params>-f mp4 -r 15 -vcodec mpeg4 -s 320x240 -b 320k -aspect 16:9 -acodec libfaac -ab 96k -ar 44100 -ac 2</params>
     <extension>mp4</extension>
     <category>Nokia</category>
   </NokiaN73N81N95WS>
   <NokiaN73N81N95FS>
     <label>Nokia Nxx Video FullScreen</label>
-    <params>-f mp4 -r 15 -vcodec mpeg4 -s 320x240 -b 320kb -aspect 4:3 -acodec libfaac -ab 96kb -ar 44100 -ac 2</params>
+    <params>-f mp4 -r 15 -vcodec mpeg4 -s 320x240 -b 320k -aspect 4:3 -acodec libfaac -ab 96k -ar 44100 -ac 2</params>
     <extension>mp4</extension>
     <category>Nokia</category>
   </NokiaN73N81N95FS>
    <MP3GP128x96mono43>
     <label>3GPP H.263 128x96 4:3 AAC mono</label>
-    <params>-r 15 -b 64kb -ac 1 -s 128x96 -ar 16000 -ab 32kb -acodec libfaac -vcodec h263</params>
+    <params>-r 15 -b 64k -ac 1 -s 128x96 -ar 16000 -ab 32k -acodec libfaac -vcodec h263</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP128x96mono43>
   <MP3GP176x144mono43>
     <label>3GPP H.263 176x144 4:3 AAC mono</label>
-    <params>-r 15 -b 64kb -ac 1 -s 176x132 -padtop 6 -padbottom 6 -ar 16000 -ab 32kb -acodec libfaac -vcodec h263</params>
+    <params>-r 15 -b 64k -ac 1 -s 176x132 -padtop 6 -padbottom 6 -ar 16000 -ab 32k -acodec libfaac -vcodec h263</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP176x144mono43>
   <MP3GP176x144mono169C>
     <label>3GPP H.263 176x144 16:9 crop AAC mono</label>
-    <params>-r 15 -b 64kb -ac 1 -s 256x144 -cropleft 40 -cropright 40 -ar 16000 -ab 32kb -acodec libfaac -vcodec h263</params>
+    <params>-r 15 -b 64k -ac 1 -s 256x144 -cropleft 40 -cropright 40 -ar 16000 -ab 32k -acodec libfaac -vcodec h263</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP176x144mono169C>
   <MP3GP176x144mono169L>
     <label>3GPP H.263 176x144 16:9 letterbox AAC mono</label>
-    <params>-r 15 -b 64kb -ac 1 -s 176x100 -padtop 22 -padbottom 22 -ar 16000 -ab 32kb -acodec libfaac -vcodec h263</params>
+    <params>-r 15 -b 64k -ac 1 -s 176x100 -padtop 22 -padbottom 22 -ar 16000 -ab 32k -acodec libfaac -vcodec h263</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP176x144mono169L>
   <MP3GP320x240stereo43>
     <label>3GPP H.264 320x240 4:3 AAC stereo</label>
-    <params>-r 15 -b 128kb -s 320x240 -ar 22050 -ab 64kb -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-r 15 -b 128k -s 320x240 -ar 22050 -ab 64k -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP320x240stereo43>
   <MP3GP320x240stereo169C>
     <label>3GPP H.264 320x240 16:9 crop AAC stereo</label>
-    <params>-r 15 -b 128kb -s 426x240 -cropleft 52 -cropright 54 -ar 22050 -ab 64kb -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-r 15 -b 128k -s 426x240 -cropleft 52 -cropright 54 -ar 22050 -ab 64k -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP320x240stereo169C>
   <MP3GP320x240stereo169L>
     <label>3GPP H.264 320x240 16:9 letterbox AAC stereo</label>
-    <params>-r 15 -b 128kb -s 320x180 -padtop 30 -padbottom 30 -ar 22050 -ab 64kb -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-r 15 -b 128k -s 320x180 -padtop 30 -padbottom 30 -ar 22050 -ab 64k -acodec libfaac -vcodec libx264 -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>3gp</extension>
     <category>Mobile Phones</category>
   </MP3GP320x240stereo169L>
    <MPEG4VideoHQ>
     <label>MPEG-4 H.264 video (high quality)</label>
-    <params>-crf 35.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 128kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 35.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 128k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mp4</extension>
     <category>MPEG-4</category>
   </MPEG4VideoHQ>
   <MPEG4VideoVHQ>
     <label>MPEG-4 H.264 video (very high quality)</label>
-    <params>-crf 25.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 160kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 25.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 160k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mp4</extension>
     <category>MPEG-4</category>
   </MPEG4VideoVHQ>
   <MPEG4VideoSHQ>
     <label>MPEG-4 H.264 video (super high quality)</label>
-    <params>-crf 15.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 192kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 15.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 192k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mp4</extension>
     <category>MPEG-4</category>
   </MPEG4VideoSHQ>
   <qmovhq>
     <label>QuickTime H.264 video (high quality)</label>
-    <params>-crf 35.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 128kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 35.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 128k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mov</extension>
     <category>QuickTime</category>
   </qmovhq>
   <qmovvhq>
     <label>QuickTime H.264 video (very high quality)</label>
-    <params>-crf 25.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 160kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 25.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 160k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mov</extension>
     <category>QuickTime</category>
   </qmovvhq>
   <qmovshq>
     <label>QuickTime H.264 video (super high quality)</label>
-    <params>-crf 15.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 192kb -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
+    <params>-crf 15.0 -vcodec libx264 -acodec libfaac -ar 48000 -ab 192k -coder 1 -flags +loop -cmp +chroma -partitions +parti4x4+partp8x8+partb8x8 -me_method hex -subq 6 -me_range 16 -g 250 -keyint_min 25 -sc_threshold 40 -i_qfactor 0.71 -b_strategy 1 -threads 0</params>
     <extension>mov</extension>
     <category>QuickTime</category>
   </qmovshq>
@@ -668,26 +668,26 @@
   </tuna-Vids>
   <BRAVIAvga>
     <label>Sony Bravia mpeg1 640x480</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg1video -s vga -b 640kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg1video -s vga -b 640k -strict -1</params>
     <extension>mpg</extension>
     <category>Bravia</category>
   </BRAVIAvga>
   <BRAVIAwide>
     <label>Sony Bravia mpeg1 640x360</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg1video -s 640x360 -b 640kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg1video -s 640x360 -b 640k -strict -1</params>
     <extension>mpg</extension>
     <category>Bravia</category>
   </BRAVIAwide>
   <BRAVIAsmall>
     <label>Sony Bravia mpeg1 320x240</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg1video -s 320x240 -b 480kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg1video -s 320x240 -b 480k -strict -1</params>
     <extension>mpg</extension>
     <category>Bravia</category>
   </BRAVIAsmall>
   <BRAVIAsmWide>
     <label>Sony Bravia mpeg1 320x180</label>
-    <params>-acodec libmp3lame -ab 128kb -ar 44100 -vcodec mpeg1video -s 320x180 -b 480kb -strict -1</params>
+    <params>-acodec libmp3lame -ab 128k -ar 44100 -vcodec mpeg1video -s 320x180 -b 480k -strict -1</params>
     <extension>mpg</extension>
     <category>Bravia</category>
   </BRAVIAsmWide>
-</presets>
\ No newline at end of file
+</presets>
